Packs/EmailCommunication/Scripts/DisplayEmailHtmlThread/DisplayEmailHtmlThread.py

```diff
@@ -55,41 +55,60 @@ def html_cleanup(full_thread_html):
     return final_html_result
 
 
+def remove_style_and_color(html_message):
+    demisto.debug(f'DisplayEmailHtmlThread - {html_message=}')
+    parsed_html_body = BeautifulSoup(html_message, 'html.parser')
+
+    # Remove all style tags
+    for style_tag in parsed_html_body.find_all('style'):
+        style_tag.decompose()
+
+    # Remove all style attributes from other tags
+    for tag in parsed_html_body.find_all(True):
```

Comment:
Let's validate the removing all the styles does not mess with how the email is forwarded/displayed in XSOAR or other platforms.
If the customer can send the email after fetching it, it will look different without the style. Remove only the color, if its the only problematic attr.